plugins/modules/x509_crl_info.py

```diff
@@ -164,99 +161,9 @@
     identify_pem_format,
 )
 
```

Comment:
The imports between `OpenSSLObjectError` and `identify_pem_format` are stale now